pkg/kubelet/status/status_manager.go

```diff
@@ -402,9 +404,21 @@ func (m *manager) updateStatusInternal(pod *v1.Pod, status v1.PodStatus, forceUp
 	}
 
 	normalizeStatus(pod, &status)
+
 	// The intent here is to prevent concurrent updates to a pod's status from
 	// clobbering each other so the phase of a pod progresses monotonically.
-	if isCached && isPodStatusByKubeletEqual(&cachedStatus.status, &status) && !forceUpdate {
+	shouldUpdate := forceUpdate
+	if !shouldUpdate && m.lastUpdateTime != nil {
+		duration := time.Now().Sub(*m.lastUpdateTime)
+		klog.V(3).Infof("it has been %v since last update", duration)
+		// if the last update is too old (more than 200 seconds ago), we need to update status
```

Comment:
s/if/If/

Reply:
Looking at existing code:
```
		// if the status has no start time, we need to set an initial time
```
I think 'if' should be good.